build.gradle.kts

```diff
@@ -101,13 +101,15 @@ val patchHistory = releaseTags
 	.map { name -> name.substring(minorTagPrefix.length) }
 
 val maxPatch = patchHistory.maxOfOrNull { it.substringBefore('-').toInt() }
-val patch = 
-	maxPatch?.plus(
-		if (patchHistory.contains(maxPatch.toString()))
-			releaseIncrement
-		else
-			0
-	) ?: 0
+val patch = maxPatch.let {
```

Comment:
Why?

Reply:
readability mostly

Reply:
This `.let` syntax bothers me for reasons I can't explain. Can you revert this and I'll just clean it up myself?